truenas/webui
# NAS-113115 / 22.02-RC.2 / Layout improvements for enclosure UI

To test:

- Go to the System-Settings > Enclosure on a supported system. 
- Make sure that populated disk trays are selectable. 

NOTE: min-width is 640px for MINIs and 700 for rackmount systems so there will be a point where horizontal scrolling kicks in.
